webpack/components/PreupgradeReportsList/components/helpers.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react';
 
+import { ListView, Badge, Icon } from 'patternfly-react';
+
 import EmptyInfoItem from './EmptyInfoItem';
 import StringInfoItem from './StringInfoItem';
 
 export const itemIteratorId = (entry, ...rest) =>
   `${entry.id}-${entry.preupgradeReportId}-${rest.join('-')}`;
 
+export const hasRemediations = entry => {
+  if (entry.includes('detail')) {
```

Comment:
`Uncaught TypeError: entry.includes is not a function`.

You can squash with the following `if` into one condition: `if (entry.detail && entry.detail.remediations)`